src/x3d-parser.js

```diff
@@ -288,7 +288,7 @@ function renderX3D(THREE, x3dXml, scene, useImageTexture) {
                 var groundGeometry = new THREE.SphereGeometry(radius, segments, segments, 0, 2 * Math.PI, 0.5 * Math.PI, 1.5 * Math.PI);
                 var groundMaterial = new THREE.MeshBasicMaterial({ fog: false, side: THREE.BackSide, vertexColors: THREE.VertexColors });
 
-                paintFaces(groundGeometry, radius, data.groundAngle, data.groundColor, false);
```

Comment:
This breaks when a `Background` tag is passed with a ground color without a ground angle. Adding a default to prevent this